GitExtensions/GitExtensions.csproj

```diff
@@ -28,21 +28,19 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="WiX" Version="$(WiXVersion)" />
- </ItemGroup>
+  </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\GitUI\GitUI.csproj" />
     <ProjectReference Include="..\GitCommands\GitCommands.csproj" />
     <ProjectReference Include="..\GitExtUtils\GitExtUtils.csproj" />
+    <ProjectReference Include="..\GitUI\GitUI.csproj" />
     <ProjectReference Include="..\Plugins\GitUIPluginInterfaces\GitUIPluginInterfaces.csproj" />
-    <ProjectReference Include="..\GitCommands\GitCommands.csproj" />
     <ProjectReference Include="..\ResourceManager\ResourceManager.csproj" />
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="app.config" />
-    <None Include="app.manifest" />
+    <None Remove="AutoCompleteRegexes.txt" />
```

Comment:
💭 I never really saw this as necessary. Including the item in both groups seems acceptable, and is less noisy.

```suggestion
```

Reply:
This is what VS writes out. I'm going to leave it as is, otherwise it would create a noise if the csproj in modified in VS.